src/main/scala/net/heartsavior/spark/sql/state/StateStoreReaderRDD.scala

```diff
@@ -47,14 +49,14 @@ class StateStoreReaderRDD(
     batchId: Long,
     operatorId: Long,
     storeName: String)
-  extends RDD[(UnsafeRow, UnsafeRow)](session.sparkContext, Nil) {
+  extends RDD[Row](session.sparkContext, Nil) {
```

Comment:
Let's update the class doc; An RDD that reads (key, value) pair of state and provides rows having columns (key, value).